HUD/IsMouseRolledOverInstructionalButtons.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: HUD
+aliases: ["0x3D9ACB1EB139E702", "_IS_MOUSE_CURSOR_ABOVE_INSTRUCTIONAL_BUTTONS"]
+---
+## IS_MOUSE_ROLLED_OVER_INSTRUCTIONAL_BUTTONS
+
+```c
+// 0x3D9ACB1EB139E702
+cs_type(Any) BOOL IS_MOUSE_ROLLED_OVER_INSTRUCTIONAL_BUTTONS();
+```
+
+Note: The buttons need to support mouse (with the TOGGLE_MOUSE_SUPPORT scaleform movie method) for it to return true.
+
+## Return value
+True if the mouse cursor is hovering above instructional buttons. 
```

Comment:
When the native name is obvious, no return value description is needed.

Reply:
Alright